README.md

```diff
@@ -1,9 +1,10 @@
-# Metropolitan Atlanta Rapid Transit Authority (MARTA) Api Client 
-[![Build Status](https://travis-ci.org/syncromatics/Syncromatics.Clients.Marta.svg?branch=master)](https://travis-ci.org/syncromatics/Syncromatics.Clients.Marta)
+# Syncromatics.Clients.Marta
 
-A .NET library to interact with the [MARTA API](http://www.itsmarta.com/app-developer-resources.aspx)
+Write a single-paragraph explanation of this project. It should describe what the software does and why it exists. Finish the paragraph with a compelling reason to use the software.
```

Comment:
@kproth, would you mind changing this to `This is a .NET library to interact with the MARTA API for getting transit arrival information in Atlanta`